dev/io.openliberty.webcontainer.servlet.6.1.internal.factories/bnd.bnd

```diff
@@ -102,4 +102,7 @@ Import-Package: \
     com.ibm.ws.resource;version=latest, \
     com.ibm.ws.kernel.boot.core;version=latest,\
     io.openliberty.jakarta.servlet.6.0;version=latest,\
```

Comment:
We don't need to point to the 6.0 API since we're already pointing to the 6.1 API below.